src/reporter/tests/utils.py

```diff
@@ -41,6 +40,47 @@ def get_notification(et, ei, attr_value, mod_value):
         ]
     }
 
+def get_notification_different_types(et, ei):
+    return [{
+        'subscriptionId': '5947d174793fe6f7eb5e39621',
+        'data': [
```

Comment:
would be better to have multiple values, instead of just 1 (for the purpose of "aggregation")

Reply:
sure, I will update my PR.